### PR TITLE
add volume konwn issue docs

### PR DESCRIPTION
**Pull request by WuJun2016**
Update v1.1

issue: https://github.com/harvester/harvester/issues/3873#issuecomment-1549303972

### CODE REVIEW

File: versioned_docs/version-v1.1/volume/create-volume.md [hunk 1]
```diff
@@ -37,4 +37,10 @@ Description: Create a volume from the Volume page.
 1. Select an existing `Image`.
 1. Configure the `Size` of the volume.
 
-![create-image-volume](/img/v1.1/volume/create-image-volume.png)
\ No newline at end of file
+![create-image-volume](/img/v1.1/volume/create-image-volume.png)
+
+## Known Issues
+
+### Volume created is not appearing in the volume list
```
Comment (vickyhella) [inline versioned_docs/version-v1.1/volume/create-volume.md:44]:
```suggestion
### The Volumes page does not show the created volume
```

File: versioned_docs/version-v1.1/volume/create-volume.md [hunk 2]
```diff
@@ -37,4 +37,10 @@ Description: Create a volume from the Volume page.
...
+
+On the volume list page, users with 'project-member' permissions cannot see the created volumes.
```
Comment (vickyhella) [inline versioned_docs/version-v1.1/volume/create-volume.md:46]:
```suggestion
**Summary**: If you are using Harvester from Rancher, after creating a volume, **project_member** users cannot find the created volume on the **Volumes** page.
**Workaround**: You can refresh the **Volumes** page and the volume should display as expected.
```

Comment (vickyhella) [reply versioned_docs/version-v1.1/volume/create-volume.md:46]: 1. Consider adding the affected versions
2. Not sure if refreshing the page or applying the `harvester-v1.1.2-patch1` is the correct workaround. If applying the patch is the workaround, please add the instructions as well
3. Consider linking to the github issue page

**Review by vickyhella — COMMENTED** [review]
Just some suggestions

**Review by LucasSaintarbor — COMMENTED** [review]
+1 suggestions made by @vickyhella

**Comment by WuJun2016** [comment]
@vickyhella   @LucasSaintarbor   PTAL again, thanks

File: versioned_docs/version-v1.1/volume/create-volume.md [hunk 3]
```diff
@@ -37,4 +37,28 @@ Description: Create a volume from the Volume page.
...
+### The Volumes page does not show the created volume
+
+**Summary**: If you are using Harvester from Rancher, after creating a volume, **project_member** users cannot find the created volume on the **Volumes** page.
+
+**Affected versions**:
+- Harvester: v1.1.2
+
+| Status    | Last updated |
+|-----------|--------------|
+| Resolved(Harvester > v1.1.2)  | 2023-07-28   |
```
Comment (vickyhella) [inline versioned_docs/version-v1.1/volume/create-volume.md:57]:
```suggestion
| Resolved (Harvester > v1.1.2)  | 2023-07-28   |
```

File: versioned_docs/version-v1.1/volume/create-volume.md [hunk 4]
```diff
@@ -37,4 +37,28 @@ Description: Create a volume from the Volume page.
...
+
+**Workaround**: You can temporarily change the Harvester plugin version to [v1.2.1-pitch1](https://github.com/harvester/dashboard/releases/tag/v1.1.2-patch1) from the Harvester UI.
+1. Go to the Harvester UI and click `Advanced`. In the `Settings` list, find ` ui-plugin-index` and then click `⋮`  > `Edit Setting`.
+2. Change the `value` to `https://releases.rancher.com/harvester-ui/plugin/harvester-release-harvester-v1.1.2-patch1/harvester-release-harvester-v1.1.2-patch1.umd.min.js`
```
Comment (vickyhella) [inline versioned_docs/version-v1.1/volume/create-volume.md:61]:
```suggestion
2. Change `Value` to `https://releases.rancher.com/harvester-ui/plugin/harvester-release-harvester-v1.1.2-patch1/harvester-release-harvester-v1.1.2-patch1.umd.min.js`.
```

File: versioned_docs/version-v1.1/volume/create-volume.md [hunk 5]
```diff
@@ -37,4 +37,28 @@ Description: Create a volume from the Volume page.
...
+3. In the `Settings` list, find `ui-source` and then click `⋮`  > `Edit Setting`.
+4. Change the `value` to `External`.
```
Comment (vickyhella) [inline versioned_docs/version-v1.1/volume/create-volume.md:63]:
```suggestion
4. Change `Value` to `External`.
```

**Review by vickyhella — APPROVED** [review]
Only some very small nits, and the rest are good to me!

**Review by vickyhella — APPROVED** [review]

File: versioned_docs/version-v1.1/volume/create-volume.md [hunk 6]
```diff
@@ -37,4 +37,28 @@ Description: Create a volume from the Volume page.
...
+
+| Status    | Last updated |
+|-----------|--------------|
+| Resolved (Harvester > v1.1.2)  | 2023-07-28   |
+
+**Workaround**: You can temporarily change the Harvester plugin version to [v1.2.1-pitch1](https://github.com/harvester/dashboard/releases/tag/v1.1.2-patch1) from the Harvester UI.
+1. Go to the Harvester UI and click `Advanced`. In the `Settings` list, find ` ui-plugin-index` and then click `⋮`  > `Edit Setting`.
+2. Change `value` to `https://releases.rancher.com/harvester-ui/plugin/harvester-release-harvester-v1.1.2-patch1/harvester-release-harvester-v1.1.2-patch1.umd.min.js`
+3. In the `Settings` list, find `ui-source` and then click `⋮`  > `Edit Setting`.
+4. Change `value` to `External`.
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.1/volume/create-volume.md:63]:
```suggestion
4. Change the **Value** to **External** to use an external UI source.
```

File: versioned_docs/version-v1.1/volume/create-volume.md [hunk 7]
```diff
@@ -37,4 +37,28 @@ Description: Create a volume from the Volume page.
...
+
+## Known Issues
+
+| Summary                                                                                                                                                                                   | Status    | Last Updated |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|--------------|
+| [The Volumes page does not show the created volume](https://github.com/harvester/harvester/issues/3874) | Resolved | 2023-07-28   |
+
+### The Volumes page does not show the created volume
+
+**Summary**: If you are using Harvester from Rancher, after creating a volume, **project_member** users cannot find the created volume on the **Volumes** page.
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.1/volume/create-volume.md:50]:
```suggestion
**Summary**: After creating a volume when using Harvester from Rancher, users with the project role **Project Member** cannot find the newly created volume on the **Volumes** page.

File: versioned_docs/version-v1.1/volume/create-volume.md [hunk 8]
```diff
@@ -37,4 +37,28 @@ Description: Create a volume from the Volume page.
...
+| Status    | Last updated |
+|-----------|--------------|
+| Resolved (Harvester > v1.1.2)  | 2023-07-28   |
+
+**Workaround**: You can temporarily change the Harvester plugin version to [v1.2.1-pitch1](https://github.com/harvester/dashboard/releases/tag/v1.1.2-patch1) from the Harvester UI.
+1. Go to the Harvester UI and click `Advanced`. In the `Settings` list, find ` ui-plugin-index` and then click `⋮`  > `Edit Setting`.
+2. Change `value` to `https://releases.rancher.com/harvester-ui/plugin/harvester-release-harvester-v1.1.2-patch1/harvester-release-harvester-v1.1.2-patch1.umd.min.js`
+3. In the `Settings` list, find `ui-source` and then click `⋮`  > `Edit Setting`.
+4. Change `value` to `External`.
+5. Log in again using the **project_member** user in Rancher.
```
Comment (LucasSaintarbor) [inline versioned_docs/version-v1.1/volume/create-volume.md:64]:
```suggestion
5. Log in again as a **Project Member** user in Rancher to view the newly created volume for your Harvester cluster. 
```

**Review by LucasSaintarbor — COMMENTED** [review]
Small nics and replaced `terms` with **terms** i.e. `volume` > **volume**

**Comment by LucasSaintarbor** [comment]
@WuJun2016 Should [docs: add ui-plugin-index #365](https://github.com/harvester/docs/pull/365) be merged before merging this PR? Lmk if the suggestions (https://github.com/harvester/docs/pull/366#pullrequestreview-1555457603) are good, and I'll merge both. Thanks!

File: versioned_docs/version-v1.1/volume/create-volume.md [hunk 9]
```diff
@@ -37,4 +37,28 @@ Description: Create a volume from the Volume page.
...
+
+**Affected versions**:
+- Harvester: v1.1.2
+
+| Status    | Last updated |
+|-----------|--------------|
+| Resolved (Harvester > v1.1.2)  | 2023-07-28   |
+
+**Workaround**: You can temporarily change the Harvester plugin version to [v1.2.1-pitch1](https://github.com/harvester/dashboard/releases/tag/v1.1.2-patch1) from the Harvester UI.
+1. Go to the Harvester UI and click `Advanced`. In the `Settings` list, find ` ui-plugin-index` and then click `⋮`  > `Edit Setting`.
```
[the remark this answers is not in the document]
Comment (WuJun2016) [reply versioned_docs/version-v1.1/volume/create-volume.md:60]: Go to to ?      @LucasSaintarbor

**Comment by WuJun2016** [comment]
> @WuJun2016 Should [docs: add ui-plugin-index #365](https://github.com/harvester/docs/pull/365) be merged before merging this PR? Lmk if the suggestions ([#366 (review)](https://github.com/harvester/docs/pull/366#pullrequestreview-1555457603)) are good, and I'll merge both. Thanks!

The two pr's are unrelated and you can merge them as much as you want.

PR comment Fix done.   only https://github.com/harvester/docs/pull/366#discussion_r1281293003 need you to confirm.

PTAL, Thanks

File: versioned_docs/version-v1.1/volume/create-volume.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (LucasSaintarbor) [reply versioned_docs/version-v1.1/volume/create-volume.md]: @WuJun2016 Fixed. Thanks for catching that

**Review by LucasSaintarbor — APPROVED** [review]
Made some formatting changes, LGTM

**Pull request merged**